### ISSUE TITLE
Make the timeslot selector always open

### ISSUE THREAD

**Issue by menor** [resmio/widget#80]
Talked to @mmarder about this. WE think that since we are always are going to have a timeslot selected, we should display the selector always open instead of it having two states.

![image](https://cloud.githubusercontent.com/assets/1360794/10278890/378e9e2e-6b61-11e5-886d-965c63700d86.png)

Also default state for the date should be collapsed cause it takes too much space right now, once we move to the just one week selector we can have it always expanded.

Maybe it'd be a good idea to make the booking info in the top panel more prominent?
![image](https://cloud.githubusercontent.com/assets/1360794/10278926/84bb2546-6b61-11e5-9ff5-8bf8422e8362.png)

We need your take on this @fabiennepimenta

**Comment by fabiennepimenta** [comment]
Yes, if we already have a time selected we don't need to show the 'Select time' 

My problem having everything 'open' or collapsed, is that turns more difficult to read. If you have a lot of information in the first view contact, it could scares the user with to much information.  

We should try to keep it clean ;) 

- Yes. I can improve the booking info.

**Comment by menor** [comment]
What about moving the booking info close to the next button? Or is it a bad idea?

**Comment by fabiennepimenta** [comment]
hm.. Something like this maybe?  Like a result of that you are doing 

![1 copy 8](https://cloud.githubusercontent.com/assets/3285116/10283874/ea336e86-6b81-11e5-9d09-0f0e433012c6.png)

**Comment by fabiennepimenta** [comment]
@menor Have you seen this? Just to not forget. 

My first version was: ( i think you have the opposite. The restaurant name and the "Online booking" is with opacity. Not the booking info. )

<img width="672" alt="screen shot 2015-10-06 at 17 03 17" src="https://cloud.githubusercontent.com/assets/3285116/10312649/621224e0-6c4c-11e5-8ecc-f6408942b6d8.png">
